Tests/UnitTests/Purchasing/OfferingsManagerTests.swift

```diff
@@ -498,6 +502,7 @@ private extension OfferingsManagerTests {
                 }
                 .dictionaryWithKeys(\.identifier),
             currentOfferingID: MockData.anyBackendOfferingsResponse.currentOfferingId,
+            placements: .init(fallbackOfferingId: "", offeringIdsByPlacement: [:]),
```

Comment:
This doesn't have the `.init(wrappedValue: [:])`, is that intentional? 

Also, since this seems to be kinda repeated, should we extract it to a constant?

Reply:
Some of these should just be `nil`! I'll go change those

But the `.init(wrappedValue: [:])` ones are only needed when initializing the `OfferingsResponse.Placements`. That one has a property wrapper in it called `@DefaultDecodable.EmptyDictionary` which is why the wrapped value is needed 🤷‍♂️ So yup yup, its intential!